velox/functions/prestosql/aggregates/ArbitraryAggregate.cpp

```diff
@@ -256,7 +256,7 @@ class NonNumericArbitrary : public exec::Aggregate {
 
 } // namespace
 
-exec::AggregateRegistrationResult registerArbitraryAggregate(
+std::vector<exec::AggregateRegistrationResult> registerArbitraryAggregate(
```

Comment:
return value doesn't seem to be used; maybe change to void

Reply:
@mbasmanova @kagamiori  I think we can make all the register functions to return void. The return type is not used. In a separate change. Let me know what you think.